src/components/specimen/components/contentBlock/OriginalData.tsx

```diff
@@ -0,0 +1,109 @@
+/* Import Dependencies */
+import DataTable, { TableColumn } from 'react-data-table-component';
+import { Row, Col, Card } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector } from 'app/hooks';
+import { getSpecimen } from "redux/specimen/SpecimenSlice";
+
+/* Import Styles */
+import styles from 'components/specimen/specimen.module.scss';
+
+
+const OriginalData = () => {
+    /* Base variables */
+    const specimen = useAppSelector(getSpecimen);
+
+    interface DataRow {
+        index: number,
+        property_name: string,
+        property_value: string,
+    };
+
+    /* Set Datatable columns */
+    const tableColumns: TableColumn<DataRow>[] = [{
+        name: 'Property',
+        selector: row => row.property_name,
+        sortable: true
+    }, {
+        name: 'Value',
+        selector: row => row.property_value,
+        sortable: true
+    }]
+
+
+    /* Set table data */
+    const tableData: DataRow[] = [];
+
+    Object.keys(specimen.originalData).forEach((property, i) => {
+        const propertyValue = specimen.originalData[property];
+
+        tableData.push({
+            index: i,
+            property_name: property,
+            property_value: propertyValue,
+        });
+    });
+
+    return (
+        <Row className="h-100">
+            <Col className="h-100">
+                <Card className="h-100">
+                    <Card.Body className="h-100">
+                        <Row className="pb-2">
+                            <Col>
+                                <Card.Title>
+                                    Original data from source
+                                </Card.Title>
+                            </Col>
+                        </Row>
+
+                        <Row className={`${styles.originalDataTable}`}>
+                            <Col md={{ span: 9 }} className="h-100">
+                                <Card className="h-100 overflow-scroll">
+                                    <DataTable
+                                        columns={tableColumns}
+                                        data={tableData}
+
+                                        striped
+                                        highlightOnHover
+                                    />
+                                </Card>
+                            </Col>
+                            <Col md={{ span: 3 }}>
+                                <Row>
+                                    <Col>
+                                        <Row>
+                                            <Col>
+                                                Harvested from:
```

Comment:
Not sure about this part.
As it currently is it is incorrect as this data doesn't come from GBIF.
If we want to keep the Harvested from we should display the URL where we collected the information.
We have that in the sourceSystem tabel.
I would remove GBIF from here and make fill this with the SourceSystemURL, I think the endpoint is already there

Reply:
https://sandbox.dissco.tech/api/v1/source-systems/prefix/suffix

Reply:
Created function to handle API call to Source Systems and replaced static content with actual representation of Source System

Reply:
Nice!